leakcanary-analyzer/src/main/java/leakcanary/LeakTrace.kt

```diff
@@ -21,17 +21,17 @@ data class LeakTrace(
     elements.dropLast(1)
         .forEachIndexed { index, leakTraceElement ->
           val currentReachability = expectedReachability[index]
-
+          val numOfSpaces = leakTraceElement.className.lastIndexOf('.') + 3
```

Comment:
Can you add a comment that explains what the `+ 3` is about?

Reply:
Hmm looks like the merge conflict didn't resolve well.

The + 3 is to include the "├─ " characters we use before printing the `leakTraceElement.className`